bulletroboy/operator.py

```diff
@@ -9,7 +9,7 @@
 from termcolor import colored
 from roboy_simulation_msgs.msg import TendonUpdate
 from geometry_msgs.msg import PoseStamped
-
+from bulletroboy.srv import HumanLinkIdFromName
```

Comment:
all messages and services will be in the roboy  communication repo. PR there with a new service type

Reply:
also it seems that this code does not work. because bulletroboy is a purely python package and it cannot contain any custom service or message definitions. you can talk to Manu or Nada, they've merged some stuff to roboy_communication already